src/Notepads/Controls/TextEditor/TextEditorCore.cs

```diff
@@ -481,13 +497,13 @@ private void SetDefaultTabStopAndLineSpacing(FontFamily font, double fontSize)
 
         private void IncreaseFontSize(double delta)
         {
-            FontSize += delta;
+            FontSize = FontSize + delta * EditorSettingsService.EditorFontSize;
         }
 
         private void DecreaseFontSize(double delta)
         {
-            if (FontSize < delta + 2) return;
-            FontSize -= delta;
+            if (FontSize < 0.11 * EditorSettingsService.EditorFontSize) return;
```

Comment:
what is this 0.11?

Reply:
I wanted the font size to only decrease when the size is greater or equal to 10% of the font size chosen in settings.